blocks/cards/cards.css

```diff
@@ -12,6 +12,24 @@
   text-align: center;
 }
 
+.section .section-container h3 {
```

Comment:
I think these styles will get applied to all `.section .section-container` . Probably you'd want `cards-container`